ansi/cursor.go

```diff
@@ -153,7 +153,7 @@ func CursorPreviousLine(n int) string {
 //	CSI n ; m H
 //
 // See: https://vt100.net/docs/vt510-rm/CUP.html
-func MoveCursor(row, col int) string {
+func MoveCursor(col, row int) string {
```

Comment:
why change the order?

Reply:
It's a common pattern to use the `x, y int` or `col, row int` than the opposite, it was confusing and led me to swap the values sometimes. It was implemented this way because the [ANSI sequence (CUP)](https://vt100.net/docs/vt510-rm/CUP.html) takes a row then a column in that order.

Reply:
Ah ok that totally makes sense. Thanks for explaining!

Reply:
Ah ok that totally makes sense. Thanks for explaining!